EndlessSkyLib.cbp

```diff
@@ -249,6 +249,7 @@
 		<Unit filename="source/PointerShader.h" />
 		<Unit filename="source/Politics.cpp" />
 		<Unit filename="source/Politics.h" />
+		<Unit filename="source/Port.h" />
```

Comment:
Why leave Port as just a header?

Reply:
What do you mean?

Reply:
Why have Port be an object that gets accessed directly through its variables instead of through functions named in a header file and implemented in a .cpp file?

Reply:
Well the class is fairly simple, so that's why I didn't bother. I can change it if you want. But like every function would be a single line (except for `Load` I guess).

Reply:
Well we're not without such classes already where everything but the Load function is just a single line (Trade is a very light class, for example), so I'd suggest just keeping with the existing style for clarity's sake. Classes that only have headers are typically reserved for template classes and enum classes.